core/src/main/scala/kafka/coordinator/group/GroupCoordinatorAdapter.scala

```diff
@@ -511,4 +532,57 @@ class GroupCoordinatorAdapter(
 
     future
   }
+
+  override def partitionFor(groupId: String): Int = {
```

Comment:
Curious about these overrides -- did the defaults not work and it was ok before because we were only using the adapter and not using it as the group coordinator?

Reply:
It works before this patch because we were still using both the adapter and the old coordinator. The goal of all this work is to put the group coordinator behind an interface so that we can swap the implementation later on. `partitionFor` is exposed by the group coordinator so we have to add it to the interface as well.

Reply:
Got it. Figured that was the case, but wanted to make sure. :)